modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * Anonymous subscriptions module.
+ */
+
+/**
+ * Implements hook_extra_field_display_info_alter().
+ */
+function oe_subscriptions_anonymous_extra_field_display_info_alter(array &$info) {
+  // This makes extra field available only for bundles where whe have flags.
+  // Get flag service.
+  $flagService = \Drupal::service('flag');
+  $flags = $flagService->getAllFlags();
+  // Compose extra field allowed bundles.
+  $bundles = [];
+  foreach ($flags as $flag) {
+    // Disabled config, nothing to do.
+    if (!$flag->status()) {
```

Comment:
We still need to check that the flag should start with subscribe_

Reply:
Done